website/docs/docs/cloud-integrations/configure-auto-exposures.md

```diff
@@ -61,9 +61,16 @@ To set up [personal access tokens (PATs)](https://help.tableau.com/current/serve
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-integration-details.jpg" title="Enter the details for the exposure connection."/>
 4. Select the collections you want to include for auto exposures. 
    
-   dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
    <Lightbox src="/img/docs/cloud-integrations/auto-exposures/cloud-select-collections.jpg" title="Select the collections you want to include for auto exposures."/>
-5. Click **Save**. 
+
+   #### Sync collections
+      :::info
+      dbt Cloud automatically imports and syncs any workbook within the selected collections. New additions to the collections will be added to the lineage in dbt Cloud during the next automatic sync (usually once per day).
+   
+      To manually trigger a sync and update collections (including additions or deletions), go to **Collections**, click **Edit**, make your checkbox selections or deselections, and then click **Save**. You should see the updates in Explorer.
```

Comment:
@joellabes @eddowh - do they have to refresh their browser in explorer or will it just show up?